helm_deploy/send-legal-mail-to-prisons/values.yaml

```diff
@@ -41,6 +41,7 @@ generic-service:
       SYSTEM_CLIENT_ID: "SYSTEM_CLIENT_ID"
       SYSTEM_CLIENT_SECRET: "SYSTEM_CLIENT_SECRET"
       SESSION_SECRET: "SESSION_SECRET"
+      BARCODE_TOKEN_PUBLIC_KEY: "BARCODE_TOKEN_PUBLIC_KEY"
```

Comment:
We are now sharing the public key with the API and so the secret name (the right hand side) is `JWT_PUBLIC_KEY`.

Reply:
So I need to change this to:
```
BARCODE_TOKEN_PUBLIC_KEY: "JWT_PUBLIC_KEY"
```
Is that right?